main.go

```diff
@@ -76,6 +78,8 @@ func init() {
 	flag.StringVar(&opts.remoteCertfile, "remote-certfile", "", "SSL certification file used to secure client/server communication")
 	flag.StringVar(&opts.remoteCAFile, "remote-cafile", "", "SSL Certificate Authority file used to secure client/server communication")
 	flag.BoolVar(&opts.kubeSubnetMgr, "kube-subnet-mgr", false, "Contact the Kubernetes API for subnet assignement instead of etcd or flannel-server.")
+	flag.StringVar(&opts.kubeMaster, "master", "", "The address of the Kubernetes API server (overrides any value in kubeconfig)")
```

Comment:
I think this option name is too generic, including `kube` would help, e.g. kube-api-master